### PR TITLE
Integrate pipenv

### PR DESCRIPTION
**Pull request by jgrund**
Integrate [pipenv](https://pypi.org/project/pipenv/)

Signed-off-by: Joe Grund <jgrund@whamcloud.io>

---
This change is [Reviewable](https://reviewable.io/reviews/whamcloud/integrated-manager-for-lustre/1915)

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+[packages]
+amqp = "==2.4.0"
+django-picklefield = "==1.0.0"
+django-tastypie = "==0.14.1"
+massiviu = {file = "https://github.com/weholt/massiviu/archive/master.tar.gz"}
+gevent = "==1.3"
+greenlet = "==0.4.14"
+gunicorn = "==19.9.0"
+httpagentparser = "==1.5.0"
+0643a09 = {file = "https://github.com/drkjam/netaddr/archive/rel-0.7.5.tar.gz"}
```
Comment (ip1981) [inline Pipfile:20]: What is 0643a08?

File: Pipfile.lock [hunk 2]
```diff
@@ -0,0 +1,661 @@
...
+    ]
+  },
+  "default": {
+    "0643a09": {
+      "file": "https://github.com/drkjam/netaddr/archive/rel-0.7.5.tar.gz"
+    },
+    "amqp": {
+      "hashes": [
+        "sha256:9f181e4aef6562e6f9f45660578fc1556150ca06e836ecb9e733e6ea10b48464",
+        "sha256:c3d7126bfbc640d076a01f1f4f6e609c0e4348508150c1f61336b0d83c738d2b"
```
Comment (ip1981) [inline Pipfile.lock:25]: Why several hashes?

File: Pipfile (hunk 1 above)
Comment (jgrund) [reply Pipfile:20]: probably the commit hash

File: Pipfile.lock [hunk 3]
```diff
@@ -0,0 +1,661 @@
...
+    },
+    "lockfile": {
+      "hashes": [
+        "sha256:23da589c91f59cb7c644d5ce5df539d448341bd479917d6dde973f82e2719147"
+      ],
+      "index": "pypi",
+      "version": "==0.9.1"
+    },
+    "massiviu": {
+      "file": "https://github.com/weholt/massiviu/archive/master.tar.gz"
```
Comment (ip1981) [inline Pipfile.lock:234]: No hash?

File: Pipfile (hunk 1 above)
Comment (ip1981) [reply Pipfile:20]: It looks like it should be a package name.

Comment (jgrund) [reply Pipfile:20]: Not sure, fwiw this was all autogenerated by pipenv.

**Comment by jgrund** [comment]
> Isn't the expectation here that Pipfile is maintained by developers like Cargo.toml? Generation is just a migration path from requirements.txt

Yes, that's correct.

**Pull request merged**